addons/sale/models/sale_order.py

```diff
@@ -864,6 +866,25 @@ def action_quotation_send(self):
             'context': ctx,
         }
 
+    def action_send_email(self):
+        """ Open a window to compose an email, with the sale order template
+        """
+        self.ensure_one()
+        report_action = self.action_quotation_send()
+        if self.env.is_admin() and not self.env.company.external_report_layout_id and not self.env.context.get('discard_logo_check'):
+            return self._action_configure_external_report_layout(report_action)
+        return report_action
+
+    def _action_configure_external_report_layout(self, report_action):
+        action = self.env["ir.actions.actions"]._for_xml_id("web.action_base_document_layout_configurator")
+        action_context = safe_eval(action['context'], {'user': self.env.user})
```

Comment:
Why you need to pass `{'user': self.env.user}`?

Reply:
Also use `literal_eval` instead of `safe_eval` (because later is giving security error)
Sorry about that :disappointed: